css_tasks/task7/index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Single Product Page</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <div class="top-bar">
+      <p class="top-bar__text">
+        start selling your products or buy them from anywhere!
+      </p>
+      <span role="button" class="top-bar__btn-close">x</span>
+    </div>
+    <header class="container header">
+      <div class="header__language">
+        <p>
+          <img
+            src="assets/usa-flag@2x.png"
+            alt=""
+            class="header__language-flag"
+          />USD<span>&#9660;</span>
+        </p>
+      </div>
+      <div class="header__logo">bonfire</div>
+      <div class="header__cart">cart(3)</div>
```

Comment:
I understand that I make same mistake on workshop but it is probably button

Reply:
Done